### PR TITLE
vhs: add page

### PR DESCRIPTION
**Pull request by GianniBYoung**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
`vhs version v0.2.0 (6f7cc81)`

### CODE REVIEW

File: pages/common/vhs.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# vhs
+
+> Generate Terminal Gifs from Code.
+> More information: <https://github.com/charmbracelet/vhs>.
+
```
Comment (kbdharun) [inline pages/common/vhs.md:5]:
```suggestion

- Create a tape file (Add commands to the tap file using your editor):

`vhs new {{path/to/file.tape}}`
```

File: pages/common/vhs.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# vhs
+
+> Generate Terminal Gifs from Code.
```
Comment (kbdharun) [inline pages/common/vhs.md:3]:
```suggestion
> CLI home video recorder to generate terminal gifs from code.
```
How about this?

File: pages/common/vhs.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://github.com/charmbracelet/vhs>.
+
+- Record inputs to a tape file. Once done, exit the shell to create the tape:
+
+`vhs record > {{tape_name}}`
```
Comment (kbdharun) [inline pages/common/vhs.md:8]:
```suggestion
`vhs record > {{path/to/file.tape}}`
```

File: pages/common/vhs.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Record inputs to a tape file using a specific shell:
+
+`vhs record --shell {{shell}} > cassette.tape`
```
Comment (kbdharun) [inline pages/common/vhs.md:12]:
```suggestion
`vhs record --shell {{shell}} > {{path/to/file.tape}}`
```

File: pages/common/vhs.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Validate a type file's syntax:
+
+`vhs validate {{tape_name_or_path}}`
```
Comment (kbdharun) [inline pages/common/vhs.md:16]:
```suggestion
`vhs validate {{path/to/file.tape}}`
```

Reference:

```bash
[dharunkrishna@dharun-fedora-laptop Videos]$ vhs validate fetch.tape
[dharunkrishna@dharun-fedora-laptop Videos]$ vhs validate
requires at least 1 arg(s), only received 0
[dharunkrishna@dharun-fedora-laptop Videos]$ cd ..
[dharunkrishna@dharun-fedora-laptop ~]$ vhs validate
requires at least 1 arg(s), only received 0
[dharunkrishna@dharun-fedora-laptop ~]$ vhs validate Videos/fetch.tape
```

File: pages/common/vhs.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Create gif from tape File:
```
Comment (kbdharun) [inline pages/common/vhs.md:18]:
```suggestion
- Create a gif from a tape file:
```

File: pages/common/vhs.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+`vhs < {{tape_name_or_path}}.tape`
```
Comment (kbdharun) [inline pages/common/vhs.md:20]:
```suggestion
`vhs < {{path/to/file.tape}}`
```
I think we can have the extension inside the placeholder, to be consistent throughout the page even though it is the only working one.

File: pages/common/vhs.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Publish a gif to vhs.charm.sh and get a shareable url:
```
Comment (kbdharun) [inline pages/common/vhs.md:22]:
```suggestion
- Publish a gif to https://vhs.charm.sh and get a shareable URL:
```

File: pages/common/vhs.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`vhs publish {{path/to/gif}}`
```
Comment (kbdharun) [inline pages/common/vhs.md:24]:
```suggestion
`vhs publish {{path/to/file.gif}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/vhs.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
+# vhs
+
+> CLI home video recorder to generate terminal gifs from code.
+> More information: <https://github.com/charmbracelet/vhs>.
+
+- Create a tape file (Add commands to the tap file using your editor):
```
Comment (kbdharun) [inline pages/common/vhs.md:6]:
```suggestion
Create a tape file (Add commands to the tap file using your editor):
```

Comment (kbdharun) [reply pages/common/vhs.md]: Sorry I accidentally made this incorrect suggestion, will fix it.

**Review by kbdharun — APPROVED** [review]
LGTM

**Review by blueskyson — APPROVED** [review]

**Pull request merged**